src/jit/emitxarch.cpp

```diff
@@ -1925,7 +1925,7 @@ UNATIVE_OFFSET      emitter::emitInsSizeAM(instrDesc* id, size_t code)
         // Most 16-bit operands will require a size prefix .
         // This refers to 66h size prefix override.
 
-        if  (    (attrSize == EA_2BYTE)
+        if  (    attrSize == EA_2BYTE
```

Comment:
Clang had an issue here with the fact that due to the rest of the condition being ifdefed out, it ended up with double parens:
```C
if ((attrSize == EA_2BYTE)) ...
```
Clang doesn't like that, since it recommends double parens for cases when there is an assignment in the condition.